lib/main.js

```diff
@@ -138,7 +139,9 @@ export default {
           '--reports=n',
           '--output-format=text',
         ];
-        if (this.rcFile !== '') {
+        if (
+          this.rcFile !== '' &&
+          fs.existsSync(`${fixPathString(this.rcFile, fileDir, projectDir)}`)) {
```

Comment:
Please put the `) {` on the next line so it doesn't look like the next line is still part of the conditions.

Reply:
Thanks, I added that